public/locales/de/translations.json

```diff
@@ -0,0 +1,49 @@
+{
+	"root.description": "Eine Alternative zu dem Nix Ökosystem",
```

Comment:
Hmm, in my mind 'Ökosystem' is pretty specific to the context of biology. But I also struggle to come up with good alternatives - maybe something like (Software-)Landschaft?

Reply:
Yeah I had the same issue. The english metaphor translates poorly into precise german. 

@christoph-heiss What do you think?

Reply:
Leaving it as it is due to lack of ideas and reviewer inactivity.